Cpp.Build.props

```diff
@@ -42,7 +42,7 @@
     <ClCompile>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <WarningLevel>Level4</WarningLevel>
-      <DisableSpecificWarnings>26800;28251;6387;4505;4702;6248;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+      <DisableSpecificWarnings>26800;28251;6387;4505;4679;4702;6248;%(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
Why did you need to disable this warning?

Reply:
This causes warnings that emit as errors with C++/CLI targeting .NET 7 for anything using `msclr.h` 

```
C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Tools\MSVC\14.35.32019\include\msclr\marshal.h(318,1): warning C4679: 'TryReadBigEndian': could not import item or its associated custom attribute (compiling source file ClassLibrary1.cpp)
1>    C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Tools\MSVC\14.35.32019\include\msclr\marshal.h(318,1): message : static virtual methods of interfaces are not supported and will be unavailable (compiling source file ClassLibrary1.cpp)
```

Reply:
@sosssego , we're adding this warning in. FYI. Please take a look later on to see if it can be removed somehow.